From b43665439e16dbc3f19bfb5f30f6c2f93734ddf3 Mon Sep 17 00:00:00 2001
From: Alejandro Sierra <b18039@freescale.com>
Date: Wed, 24 Oct 2012 15:32:37 -0500
Subject: [PATCH 2201/2463] ENGR00231063 Sabreauto: fix share pins SD Card and
 NFC

SD card card detection and NFC controller CS2 share
the same pin on ARD platform. However CS2 is not
connected to the socket. This signal was removed
from the sabreauto board file.

Signed-off-by: Alejandro Sierra <b18039@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabreauto.h    |    2 --
 arch/arm/mach-mx6/board-mx6solo_sabreauto.h |    2 --
 2 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabreauto.h b/arch/arm/mach-mx6/board-mx6q_sabreauto.h
index 88cb7ff..fee469a 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabreauto.h
+++ b/arch/arm/mach-mx6/board-mx6q_sabreauto.h
@@ -257,8 +257,6 @@ static iomux_v3_cfg_t mx6q_gpmi_nand[] __initdata = {
 	MX6Q_PAD_NANDF_ALE__RAWNAND_ALE,
 	MX6Q_PAD_NANDF_CS0__RAWNAND_CE0N,
 	MX6Q_PAD_NANDF_CS1__RAWNAND_CE1N,
-	MX6Q_PAD_NANDF_CS2__RAWNAND_CE2N,
-	MX6Q_PAD_NANDF_CS3__RAWNAND_CE3N,
 	MX6Q_PAD_NANDF_RB0__RAWNAND_READY0,
 	MX6Q_PAD_SD4_DAT0__RAWNAND_DQS,
 	MX6Q_PAD_NANDF_D0__RAWNAND_D0,
diff --git a/arch/arm/mach-mx6/board-mx6solo_sabreauto.h b/arch/arm/mach-mx6/board-mx6solo_sabreauto.h
index b752faa..18f5d90 100644
--- a/arch/arm/mach-mx6/board-mx6solo_sabreauto.h
+++ b/arch/arm/mach-mx6/board-mx6solo_sabreauto.h
@@ -254,8 +254,6 @@ static iomux_v3_cfg_t mx6dl_gpmi_nand[] __initdata = {
 	MX6DL_PAD_NANDF_ALE__RAWNAND_ALE,
 	MX6DL_PAD_NANDF_CS0__RAWNAND_CE0N,
 	MX6DL_PAD_NANDF_CS1__RAWNAND_CE1N,
-	MX6DL_PAD_NANDF_CS2__RAWNAND_CE2N,
-	MX6DL_PAD_NANDF_CS3__RAWNAND_CE3N,
 	MX6DL_PAD_NANDF_RB0__RAWNAND_READY0,
 	MX6DL_PAD_SD4_DAT0__RAWNAND_DQS,
 	MX6DL_PAD_NANDF_D0__RAWNAND_D0,
-- 
1.7.7.4

